index.html

```diff
@@ -5,17 +5,144 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <!-- Add a link to your custom CSS here -->
+    <link rel="stylesheet" href="./css/common.css">
+    <link rel="stylesheet" href="./css/main.css">
+     <script src="https://kit.fontawesome.com/9569a39f56.js" crossorigin="anonymous"></script>
 
     <title>Responsive grid project</title>
 </head>
 
 <body>
 
+    <!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <!-- Add a link to your custom CSS here -->
+    <link rel="stylesheet" href="./css/main.css" />
+    <script
+      src="https://kit.fontawesome.com/9569a39f56.js"
+      crossorigin="anonymous"
+    ></script>
+    <title>Responsive grid project</title>
+  </head>
+​
+  <body>
     <!-- Add your markup here -->
-
-
+​
+    <div class="grid-container">
+      <header id="header">
+        <section class="navbar">
+          <ul>
+            <li class="name">
+              <b> <i id="square" class="fas fa-square"></i> Maya Nelson</b>
+            </li>
+            <li>PROJECT MANAGER</li>
+          </ul>
+          <a href="about.html">
+            <img src="hamburger.png" class="hamburger" alt="align-icon"
+          /></a>
+​
+          <nav id="nav">
+            <ul class="lists">
+              <li>About</li>
+              <li>Resume</li>
+              <li>Projects</li>
+              <li>Contacts</li>
+            </ul>
+          </nav>
+        </section>
+      </header>
+​
+     <main>
+        <div class="my-image">
```

Comment:
To put an absolute image in the center, you can have a look at this answer: https://stackoverflow.com/a/8273750/1189105